### PR TITLE
Yosabet plant

### PR DESCRIPTION
**Pull request by YosabetK**

### CODE REVIEW

**Review by Chronowork1 — APPROVED** [review]

File: migrations/20210422022213-create-profile.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Profiles', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userId: {
```
Comment (francisli) [inline migrations/20210422022213-create-profile.js]: Foreign keys should be CamelCase capitalized...  UserId not userId ...

File: migrations/20210422022213-create-profile.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      location: {
+        type: Sequelize.STRING
+      },
+      descrption: {
```
Comment (francisli) [inline migrations/20210422022213-create-profile.js]: Typo: description

File: migrations/20210422022213-create-profile.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        type: Sequelize.STRING
+      },
+      hour: {
+        type: Sequelize.DATEONLY
+      },
+      phonenumber: {
+        type: Sequelize.INTEGER
```
Comment (francisli) [inline migrations/20210422022213-create-profile.js]: Phone numbers, stored as STRING

File: migrations/20210422022213-create-profile.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      location: {
+        type: Sequelize.STRING
+      },
+      descrption: {
+        type: Sequelize.STRING
+      },
+      hour: {
+        type: Sequelize.DATEONLY
+      },
+      phonenumber: {
```
Comment (francisli) [inline migrations/20210422022213-create-profile.js]: Recommend: phoneNumber

File: models/attendees.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Attendees extends Model {
```
Comment (francisli) [inline models/attendees.js]: The model class name should be singular Attendee, not plural Attendees

File: models/attendees.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  };
+  Attendees.init({
```
Comment (francisli) [inline models/attendees.js]: Singular

File: models/attendees.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    GardenId: DataTypes.INTEGER,
+    name: DataTypes.STRING,
+    email: DataTypes.STRING,
+    phoneNumber: DataTypes.INTEGER,
```
Comment (francisli) [inline models/attendees.js]: Use STRING

File: models/attendees.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    date: DataTypes.DATE,
+    hour: DataTypes.DATEONLY
+  }, {
+    sequelize,
+    modelName: 'Attendees',
```
Comment (francisli) [inline models/attendees.js]: Singular

File: models/attendees.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+  });
+  return Attendees;
```
Comment (francisli) [inline models/attendees.js]: Singular

File: models/attendees.js [hunk 10]
```diff
@@ -0,0 +1,28 @@
+'use strict';
```
Comment (francisli) [inline models/attendees.js]: Please rename this file from models/attendees.js to models/attendee.js so that the filename is singular as well.

**Review by francisli — CHANGES_REQUESTED** [review]

**Pull request merged**